blockchain/blockchain.go

```diff
@@ -231,51 +233,6 @@ func NewBlockchain(cfg *config.Config, opts ...Option) Blockchain {
 	if chain.sf != nil {
 		chain.lifecycle.Add(chain.sf)
 	}
-	if err := chain.Start(context.Background()); err != nil {
```

Comment:
good clean-up to separate Start() from New(), this makes it consistent

Reply:
ps. i had a similar change some time ago to split Start() from New() for SF and Trie, at that time noticed also need to do for Blockchain. Good to have this PR, makes codebase consistent